### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by panll**
update with NETCDF added

### CODE REVIEW

File: docs/UsersGuide/source/glossary.rst [hunk 1]
```diff
@@ -48,6 +48,9 @@ Glossary
    NEMSIO
       A binary format for atmospheric model output on the native gaussian grid.
 
+   NETCDF
+      A set of software libraries and machine-independent data formats that support the creation, access, and sharing of array-oriented scientific data.
```
Comment (JulieSchramm) [inline docs/UsersGuide/source/glossary.rst:52]: Change support to supports.

**Comment by panll** [comment]
It's updated. Thanks, Julie!

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 2]
```diff
@@ -142,19 +143,24 @@ Initial condition formats and source
...
+  These files cover the entire globe down to a horizontal resolution of 13 km and
+  can be found at the FTP data repository `<https://ftp.emc.ncep.noaa.gov/EIB/UFS/>`_.  
+     
 - **GRIB2**
 
   These files cover the entire globe and resolutions of 0.5, or 1.0 degree are supported.
 
-  - 0.5 deg files are available at `<https://nomads.ncdc.noaa.gov/data/gfs4/>`_
+  - 0.5 deg files are available at `<https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g4-anl-files-old/catalog.html>`_
   - 1.0 deg files can be requested from `<https://www.ncdc.noaa.gov/has/HAS.FileAppRouter?datasetname=GFS3&subqueryby=STATION&applname=&outdest=FILE>`_
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:164]:
```suggestion
  - 1.0 deg files can be requested from `<https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g3-anl-files-old/catalog.html>`_
```
Isn't is best to point the user to the THEDDS data server for all types of GRIB2 files?

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 3]
```diff
@@ -166,15 +172,19 @@ The default naming convention for the initial conditions files is described belo
...
+  - Three-dimensional atmosphere state ``gfs.tHHz.atmanl.nemsio`` 
+
+- **NETCDF**
 
+  - Two-dimensional surface variables ``gfs.tHHz.sfcanl.nc``
+  - Three-dimensional atmosphere state ``gfs.tHHz.atmanl.nc`` 
+ 
 - **GRIB2**
 
   - Surface variables and atmosphere state ``gfsanl_4_YYYYMMDD_HH00_000.grb2``
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:184]:
```suggestion
  - Surface variables and atmosphere state in 0.5 deg ``gfsanl_4_YYYYMMDD_HH00_000.grb2``
```

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 4]
```diff
@@ -311,12 +321,27 @@ The data should be placed in ``$DIN_LOC_IC``.
...
          ./get.sh 20191224 12
 
-If the file is a gfs3 file not a gfs4 file, the user must link the new file to the old file name. For example, 
+     For downloading grib2 files, the same code ``get.sh`` can be used except replacing wget part by the following line: 
+
+     .. code-block:: console
+
+         wget -c https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g4-anl-files/$yyyymmdd/gfs_4_${yyyymmdd)_${hh}00_000.grb2
+
+     If the file is a gfs3 file not a gfs4 file, the user must link the new file to the old file name. For example,
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:330]:
```suggestion
     For downloading files in GRIB2 format with 1.0 degree grid spacing, the same code ``get.sh`` can be used except the wget command should be replaced with the following line: 
     
     .. code-block:: console

         wget -c https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g3-anl-files/$yyyymmdd/gfs_3_${yyyymmdd)_${hh}00_000.grb2
     
     If the file has 1.0 degree resolution (gfs3 file), the user must link the new file to the name expected by the App. For example, 
```
It is possible that this linking bug will be fixed before the release. If so, we'll update this text later.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 5]
```diff
@@ -311,12 +321,27 @@ The data should be placed in ``$DIN_LOC_IC``.
...
          ln -s gfs_3_20190829_0000_000.grb2 gfs_4_20190829_0000_000.grb2
 
+     For downloading NETCDF files, the wget parts in ``get.sh`` need to be changed to:
+
+     .. code-block:: console
+
+         wget -c https://ftp.emc.ncep.noaa.gov/EIB/UFS/inputdata/$yyyymm/gfs.$yyyymmdd/$hh/gfs.t${hh}z.atmf000.nc
+         wget -c https://ftp.emc.ncep.noaa.gov/EIB/UFS/inputdata/$yyyymm/gfs.$yyyymmdd/$hh/gfs.t${hh}z.sfcf000.nc
+
+     Currently, only a few sample NETCDF files are available for testing at the FTP data repository.
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:343]:
```suggestion
     Currently, only two sample netCDF files are available for testing at the FTP data repository.
```

File: docs/UsersGuide/source/introduction.rst [hunk 6]
```diff
@@ -31,16 +31,16 @@ Pre-processor and initial conditions
 The MR Weather App is distributed with the :term:`chgres_cube` pre-processing software.
 It converts the Global Forecast System (GFS) analyses to the format needed as
 input to the model, which is six tiles in NetCDF format. Additional information
-about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/ufs-v1.1.0/>`_.
+about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/latest/>`_.
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/introduction.rst:34]:
```suggestion
about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/ufs-v1.1.0/>`_.
```

File: docs/UsersGuide/source/introduction.rst [hunk 7]
```diff
@@ -31,16 +31,16 @@ Pre-processor and initial conditions
...
+v2 (:term:`GRIB2`) format (in 0.25 , 0.50, or 1.0 degree grid spacing),  the NOAA Environmental
+Modeling System (:term:`NEMS`) Input/Output (:term:`NEMSIO`) format, or Network Common Data Formt (:term:`NETCDF`).
 Initialization from dates starting on January 1, 2018 are supported. Dates
 before that may work, but are not guaranteed. GFS public archives can be
-accessed through the `National Centers for Environmental Information (NCEI)
-<https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_
-or through the `NOAA Operational Model Archive and Distribution System (NOMADS) <https://nomads.ncep.noaa.gov/>`_ .
+accessed through the National Centers for Environmental Information (NCEI)
+`Global Forecast System website <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_
+or through the `Thredds Data Server at NCEI <https://www.ncei.noaa.gov/thredds/model/gfs.html>`_. Some NETCDF data can be found at `the EMC UFS ftp site <https://ftp.emc.ncep.noaa.gov/EIB/UFS/>`_.
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/introduction.rst:43]:
```suggestion
accessed through the `THREDDS Data Server at NCEI <https://www.ncei.noaa.gov/thredds/model/gfs.html>`_. A small sample of netCDF files can be found at `the EMC FTP site <https://ftp.emc.ncep.noaa.gov/EIB/UFS/>`_.
```

File: docs/UsersGuide/source/introduction.rst [hunk 8]
```diff
@@ -116,7 +116,7 @@ producing reasonable results.
 
 The scripts are available in the ftp site ftp://ftp.emc.ncep.noaa.gov/EIB/UFS/visualization_example/.
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/introduction.rst:117]:
```suggestion
The scripts are available in the FTP site ftp://ftp.emc.ncep.noaa.gov/EIB/UFS/visualization_example/.
```

File: docs/UsersGuide/source/glossary.rst [hunk 9]
```diff
@@ -48,6 +48,9 @@ Glossary
    NEMSIO
       A binary format for atmospheric model output on the native gaussian grid.
 
+   NETCDF
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/glossary.rst:51]: netCDF?

File: docs/UsersGuide/source/faq.rst [hunk 10]
```diff
@@ -411,7 +411,7 @@ throughout the forecast. Compset GFSv15p2 can use either the GFSv15p2 suite or
 the GFSv15p2_no_nsst suite. Similarly, Compset GFSv16beta can use either the
 GFSv16beta suite or the GFSv16beta_no_nsst suite. The choice is made based on the
 format of the initial conditions file. When GRIB2 format is chosen, the non_nsst
-suites are used. When NEMSIO format is chosen, the suites with NSST are chosen.
+suites are used. When NEMSIO or NETCDF format is chosen, the suites with NSST are chosen.
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/faq.rst]: For consistency, do we need to use NetCDF everywhere?

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 11]
```diff
@@ -311,12 +321,27 @@ The data should be placed in ``$DIN_LOC_IC``.
          chmod 755 get.sh
          ./get.sh 20191224 12
 
-If the file is a gfs3 file not a gfs4 file, the user must link the new file to the old file name. For example, 
+     For downloading grib2 files, the same code ``get.sh`` can be used except replacing wget part by the following line:
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 12]
```diff
@@ -311,12 +321,27 @@ The data should be placed in ``$DIN_LOC_IC``.
...
+
+         wget -c https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g4-anl-files/$yyyymmdd/gfs_4_${yyyymmdd)_${hh}00_000.grb2
+
+     If the file is a gfs3 file not a gfs4 file, the user must link the new file to the old file name. For example, 
 
      .. code-block:: console
 
          ln -s gfs_3_20190829_0000_000.grb2 gfs_4_20190829_0000_000.grb2
 
+     For downloading NETCDF files, the wget parts in ``get.sh`` need to be changed to:
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 13]
```diff
@@ -166,15 +172,19 @@ The default naming convention for the initial conditions files is described belo
 - **NEMSIO**
 
   - Two-dimensional surface variables ``gfs.tHHz.sfcanl.nemsio``
-  - Three-dimensional atmosphere state ``gfs.tHHz.atmanl.nemsio``
+  - Three-dimensional atmosphere state ``gfs.tHHz.atmanl.nemsio`` 
+
+- **NETCDF**
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 14]
```diff
@@ -142,19 +143,24 @@ Initial condition formats and source
...
+The GFS data can be provided in three formats: :term:`NEMSIO`, :term:`NETCDF`, or :term:`GRIB2`. Both types of files can be obtained
 from the `NCEI website <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_.
 
 - **NEMSIO**
 
   These files cover the entire globe down to a horizontal resolution of 13 km and
-  can be found at `<https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/>`_.
+  can be found at `<https://nomads.ncep.noaa.gov/pub/data/nccf/com/gfs/prod/>`_.  
+  
+- **NETCDF**
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 15]
```diff
@@ -142,19 +143,24 @@ Initial condition formats and source
 
 The UFS MR Weather App currently only supports the use of Global Forecast System
 (GFS) data as raw initial conditions (that is, MRF, AVN, ERA5 etc. are not supported).
-The GFS data can be provided in two formats: NEMSIO or GRIB2. Both types of files can be obtained
+The GFS data can be provided in three formats: :term:`NEMSIO`, :term:`NETCDF`, or :term:`GRIB2`. Both types of files can be obtained
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 16]
```diff
@@ -14,10 +14,11 @@ Input files
...
 run the end-to-end system and write output files to disk. Depending on the dates and format
-(`GRIB2 <https://www.nco.ncep.noaa.gov/pmb/docs/grib2/>`_ and
-`NEMSIO <https://github.com/NOAA-EMC/NCEPLIBS-nemsio/wiki/Home-NEMSIO>`_)
-requested, input files can be automatically retrieved by CIME (GRIB2) or must be staged by
-the user (:term:`NEMSIO`).
+(`GRIB2 <https://www.nco.ncep.noaa.gov/pmb/docs/grib2/>`_,
+`NEMSIO <https://github.com/NOAA-EMC/NCEPLIBS-nemsio/wiki/Home-NEMSIO>`_, or 
+`NETCDF <https://www.unidata.ucar.edu/software/netcdf/>`_)
+requested, input files can be automatically retrieved by CIME (:term:`GRIB2`) or must be staged by
+the user (:term:`NEMSIO` or :term:`NETCDF`).
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Updated.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 17]
```diff
@@ -14,10 +14,11 @@ Input files
 
 The :term:`UFS` MR Weather App requires numerous input files. :term:`CIME` can copy/link to input files,
 run the end-to-end system and write output files to disk. Depending on the dates and format
-(`GRIB2 <https://www.nco.ncep.noaa.gov/pmb/docs/grib2/>`_ and
-`NEMSIO <https://github.com/NOAA-EMC/NCEPLIBS-nemsio/wiki/Home-NEMSIO>`_)
-requested, input files can be automatically retrieved by CIME (GRIB2) or must be staged by
-the user (:term:`NEMSIO`).
+(`GRIB2 <https://www.nco.ncep.noaa.gov/pmb/docs/grib2/>`_,
+`NEMSIO <https://github.com/NOAA-EMC/NCEPLIBS-nemsio/wiki/Home-NEMSIO>`_, or 
+`NETCDF <https://www.unidata.ucar.edu/software/netcdf/>`_)
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated.

File: docs/UsersGuide/source/glossary.rst (hunk 9 above)
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/glossary.rst]: Updated.

File: docs/UsersGuide/source/faq.rst [hunk 18]
```diff
@@ -455,7 +455,7 @@ task for NCEP-Post, the subgroup option needs to set to ``case.gfs_post``.
 How to change the filenames for input to chgres_cube?
 =====================================================
 
-By default, CIME uses `pre-defined convention <https://ufs-mrweather-app.readthedocs.io/en/latest/inputs_outputs.html>`_ to define directory and file names for raw input to ``chgres_cube``. In this case, 0.5-degree data in GRIB2 format is used from `NCDC - Global Forecast System <https://www.ncdc.noaa.gov/data-access/model-data/model-datasets/global-forcast-system-gfs>`_.
+By default, CIME uses `pre-defined convention <https://ufs-mrweather-app.readthedocs.io/en/latest/inputs_outputs.html>`_ to define directory and file names for raw input to ``chgres_cube``. In this case, 0.5-degree data in GRIB2 format is used from `NCEI - THREDDS DATA Server <https://www.ncei.noaa.gov/thredds/model/gfs.html>`_.
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/faq.rst]: Udated.

File: docs/UsersGuide/source/faq.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/faq.rst]: Updated.

File: docs/UsersGuide/source/inputs_outputs.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Updated.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 19]
```diff
@@ -331,14 +356,15 @@ Directory `$DIN_LOC_IC/YYMMMM/YYYYMMDD`` can have both GRIB2 and NEMSIO files fo
 a given initialization hour and can have files for multiple initialization hours
 (00, 06, 12, and 18 UTC).
 
-If a directory has both GRIB2 and NEMSIO files for the same initialization date and time,
+If a directory has both GRIB2 and NEMSIO (or NETCDF) files for the same initialization date and time,
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: updated.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 20]
```diff
@@ -331,14 +356,15 @@ Directory `$DIN_LOC_IC/YYMMMM/YYYYMMDD`` can have both GRIB2 and NEMSIO files fo
...
 CIME will use the GRIB2 files. If the user wants to change this behavior so CIME uses the
-NEMSIO files, the user should edit file ``user_nl_ufsatm``
+NEMSIO or NETCDF files, the user should edit file ``user_nl_ufsatm``
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Updated.

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 21]
```diff
@@ -331,14 +356,15 @@ Directory `$DIN_LOC_IC/YYMMMM/YYYYMMDD`` can have both GRIB2 and NEMSIO files fo
...
 and add
 
 .. code-block:: console
 
-    input_type = "gaussian"
+    input_type = "gaussian" for NEMSIO
+    input_type = "gaussian_netcdf" for NETCDF
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Changed.

File: docs/UsersGuide/source/introduction.rst [hunk 22]
```diff
@@ -31,16 +31,16 @@ Pre-processor and initial conditions
...
 It converts the Global Forecast System (GFS) analyses to the format needed as
 input to the model, which is six tiles in NetCDF format. Additional information
-about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/ufs-v1.1.0/>`_.
+about chgres_cube can be found in the `chgres_cube User’s Guide <https://ufs-utils.readthedocs.io/en/latest/>`_.
 
 GFS analyses for initializing the MR Weather App can be in Gridded Binary
-v2 (GRIB2) format (in 0.25 , 0.50, or 1.0 degree grid spacing) or in the NOAA Environmental
-Modeling System (:term:`NEMS`) Input/Output (:term:`NEMSIO`) format.
+v2 (:term:`GRIB2`) format (in 0.25 , 0.50, or 1.0 degree grid spacing),  the NOAA Environmental
+Modeling System (:term:`NEMS`) Input/Output (:term:`NEMSIO`) format, or Network Common Data Formt (:term:`NETCDF`).
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: Changed, thanks!

File: docs/UsersGuide/source/introduction.rst [hunk 23]
```diff
@@ -60,16 +60,16 @@ provides more information about FV3 and its grids. Additional information about
...
 Two of them are variations of an updated version of the physics :term:`suite` used in
 the operational GFS v15, while the other two are variations of an experimental
 suite that includes a subset of the developments for the next version of GFS,
 GFS v16. The variations pertain to how the sea surface temperature (SST) is
 initialized and parameterized to evolve, and are chosen depending on the type
-of initial conditions for the App. Initial conditions in GRIB2 format have a
+of initial conditions for the App. Initial conditions in :term:`GRIB2` format have a
 single two-dimensional field to initialize the SST, which must be kept constant
-throughout the forecast. Initial conditions in :term:`NEMSIO` format have two two-dimensional
+throughout the forecast. Initial conditions in :term:`NEMSIO` or :term:`NETCDF` format have two two-dimensional
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: Changed.

File: docs/UsersGuide/source/introduction.rst [hunk 24]
```diff
@@ -97,7 +97,7 @@ Post-processor
 
 The MR Weather App is distributed with a post-processing tools, the Unified
 Post Processor (UPP). The Unified Post Processor (UPP) converts the
-native NetCDF output from the model to the GRIB2 format on standard isobaric
+native NetCDF output from the model to the :term:`GRIB2` format on standard isobaric
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: Changed.

File: docs/UsersGuide/source/introduction.rst [hunk 25]
```diff
@@ -116,7 +116,7 @@ producing reasonable results.
 
 The scripts are available in the ftp site ftp://ftp.emc.ncep.noaa.gov/EIB/UFS/visualization_example/.
 File visualization_README describes the plotting scripts. Example plots are provided
-for the C96 5-day forecasts initialized on 8/29/2019 00 UTC using GRIB2 and NEMSIO
+for the C96 5-day forecasts initialized on 8/29/2019 00 UTC using :term:`GRIB2`,  :term:`NEMSIO`, or :term:`NETCDF`
```
[the remark this answers is not in the document]
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: Changed.

File: docs/UsersGuide/source/inputs_outputs.rst (hunk 2 above)
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst:164]: Changed, thanks!

File: docs/UsersGuide/source/inputs_outputs.rst (hunk 3 above)
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst:184]: Changed, thanks!

File: docs/UsersGuide/source/inputs_outputs.rst (hunk 4 above)
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst:330]: Changed, thanks!

File: docs/UsersGuide/source/inputs_outputs.rst (hunk 5 above)
Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst]: Changed.

File: docs/UsersGuide/source/introduction.rst (hunk 6 above)
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: Changed.

File: docs/UsersGuide/source/introduction.rst (hunk 7 above)
Comment (panll) [reply docs/UsersGuide/source/introduction.rst]: changed

File: docs/UsersGuide/source/introduction.rst (hunk 8 above)
Comment (panll) [reply docs/UsersGuide/source/introduction.rst:117]: Changed, thanks!

**Comment by ligiabernardet** [comment]
Yes. Please use netCDF if it is in the middle of a sentence, and NetCDF at
the beginning of sentences.

**Comment by panll** [comment]
Great, thanks!

File: docs/UsersGuide/source/inputs_outputs.rst [hunk 26]
```diff
@@ -161,7 +161,7 @@ from the `NCEI website <https://www.ncdc.noaa.gov/data-access/model-data/model-d
   These files cover the entire globe and resolutions of 0.5, or 1.0 degree are supported.
 
   - 0.5 deg files are available at `<https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g4-anl-files-old/catalog.html>`_
-  - 1.0 deg files can be requested from `<https://www.ncdc.noaa.gov/has/HAS.FileAppRouter?datasetname=GFS3&subqueryby=STATION&applname=&outdest=FILE>`_
+  - 1.0 deg files can be requested from `<https://www.ncei.noaa.gov/hthredds/catalog/model-gfs-g3-anl-files-old/catalog.html`_
```
Comment (ligiabernardet) [inline docs/UsersGuide/source/inputs_outputs.rst:164]:
```suggestion
  - 1.0 deg files can be requested from `<https://www.ncei.noaa.gov/thredds/catalog/model-gfs-g3-anl-files-old/catalog.html`_
```

Comment (panll) [reply docs/UsersGuide/source/inputs_outputs.rst:164]: Changed, thanks!

**Comment by panll** [comment]
Sure, thanks! I'll double check the "netCDF" part.

**Comment by panll** [comment]
Changed "NetCDF" to "netCDF", thanks!

**Comment by ligiabernardet** [comment]
@climbfuji Could you review/merge these changes in so we can work on the last set of documentation updates/changes now that the mechanism to stage/ingest data is finalized?

**Pull request merged**